lesson9.tex

```diff
@@ -0,0 +1,123 @@
+                 made their decisions before going to the next round)
+        \item All stocks prices should move in each round, to simulate the stock market's general trend of upwards growth
+        \item Students have the opportunity to sell their stocks and buy from the Bot Island Store during any round. 
+        \item Instructors should make a point of talking about how their stocks are doing, since each will be
+                 following a different strategy. The instructor who invested all their money in one stock should be 
+                 exaggeratedly happy when the prices go up, and devastated when it goes down. The instructor
+                 who evenly diversified should be a little jealous when the stock of the company the other instructor 
+                 invested in rises, and be relieved when their stocks aren't too negatively affected in the event of a negative event. 
+      \end{itemize}
+    \subsection{Extensions}
+      \begin{itemize}
+        \item After every third round, students can receive dividends from their stocks, which will be added BotCash 
+                 that the students can reinvest, or keep with them.
+        \item Give students "balance sheets" of the companies containing only the most relevant information, 
+                 like how much they return in dividends, their revenue, how many stocks they have in the market, etc
+      \end{itemize}
+    \subsection{Conclusion and Evaluation}
+      At the end of the last round, announce that we have all gone into retirement, and we are now going to review
+      and sell our stocks. Ask students how well they fared, and see if we can find who did the best. The student
+      who did the best will discuss their strategy, and we will discuss as a class if we believe this strategy would
+      work every time. Then, open the discussion to other students to see what kinds of strategies they used.
+      Ask them if they have any opinions about how an instructor did. Once students have come to a conclusion,
+      show data about how different funds fare, and emphasize how index funds produce the best results over
+      time. Explain the concept of compound interest, and how it can take a small amount of money you have
+      and grow it over time. Emphasize this fact by displaying a compounding calculator, and showing how 
+      investing earlier vs later is best (and how much money they can earn, even with seemingly small rates).
+      Finally, give students the amount of cash they've earned (possibly in forms of checks if we don't have
+      enough paper money), and allow them to buy what they want at the Bot Island store.
+      They go home with cool stuff and a knowledge of the stock market!
+      \subsubsection{Exit Questions}
+      \begin{itemize}
+        \item What is the stock market, and what is the best strategy for investing in the market?
```

Comment:
Maybe not so broad, there is no "best" strategy, and we should be aiming to teach them the basics of investing, and show how parts of it is predictable, but other parts are not as much.

Reply:
I amended this to make it several strategies, and I am assuming that this question will be answered when kids begin comparing what strategies they used